tests/MoBi.Tests/Core/SBML/ContextForSBMLIntegration.cs

```diff
@@ -23,7 +23,7 @@ protected override void Context()
          var context = IoC.Resolve<IMoBiContext>();
          context.NewProject();
          _moBiProject = context.CurrentProject;
-         context.LoadFrom(_moBiProject);
+         //context.LoadFrom(_moBiProject);
```

Comment:
why this change

Reply:
I am not sure why this line is needed but I have just restored it. I was experimenting a bit and forgot to restore it on my last commit.